tests/unit/test_propose_downstream.py

```diff
@@ -65,21 +63,18 @@
     ],
 )
 def test_branches(jobs, job_config_trigger_type, branches_override, branches):
+    project = flexmock(
+        default_branch="main",
+    )
+    service_config = ServiceConfig().get_service_config()
+    flexmock(ServiceConfig, get_project=lambda url: project)
     propose_downstream_helper = ProposeDownstreamJobHelper(
-        service_config=ServiceConfig(),
+        service_config=service_config,
```

Comment:
Is this `service_config` instantiation here for some reason?

Reply:
fixed, thanks!

Reply:
fixed, thanks!